bindings/rust-examples/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = [
-    "client-hello-config-resolution", "tokio-server-client",
+    "client-hello-config-resolution", "hyper-server-client", "tokio-server-client",
```

Comment:
Nit: we should probably list these vertically rather than horizontally before this gets more out of hand